tests/Repositories/ProjectRepositoryTest.php

```diff
@@ -90,22 +90,40 @@ public function it_can_retrieve_projects_of_logged_in_user()
     }
 
     /** @test */
-    public function it_can_retrieve_projects_list_array_of_logged_in_user()
+    public function it_can_retrieve_projects_list_array_when_logged_in_user_has_manage_projects_permission()
     {
         $mitul = factory(User::class)->create();
         $project = factory(Project::class)->create();
         $project->users()->sync([$mitul->id]);
 
-        $projectsOfLoggedInUser = factory(Project::class)->create();
-        $projectsOfLoggedInUser->users()->sync([$this->defaultUserId]);
+        $projectOfLoggedInUser = factory(Project::class)->create();
+        $projectOfLoggedInUser->users()->sync([$this->defaultUserId]);
 
-        $myProjects = $this->projectRepo->getLoginUserAssignProjectsArr();
-        $this->assertCount(1, $myProjects);
+        $allProjects = $this->projectRepo->getLoginUserAssignProjectsArr();
 
-        $totalProjects = $this->projectRepo->getProjectsList();
-        $this->assertCount(2, $totalProjects);
+        $this->assertCount(2, $allProjects);
+        $this->assertArrayHasKey($projectOfLoggedInUser->id, $allProjects);
+        $this->assertArrayHasKey($project->id, $allProjects);
+        $this->assertContains($projectOfLoggedInUser->name, $allProjects);
+        $this->assertContains($project->name, $allProjects);
+    }
+
+    /** @test */
+    public function it_can_retrieve_assigned_projects_list_array_of_logged_in_user()
+    {
+        $project = factory(Project::class)->create();
+        $project->users()->sync([$this->defaultUserId]);
+
+        $mitul = factory(User::class)->create();
+        $projectOfLoggedInUser = factory(Project::class)->create();
+        $projectOfLoggedInUser->users()->sync([$mitul->id]);
+        $this->actingAs($mitul);
+
+        $allProjects = $this->projectRepo->getLoginUserAssignProjectsArr();
 
-        $this->assertArrayHasKey($projectsOfLoggedInUser->id, $myProjects);
-        $this->assertContains($projectsOfLoggedInUser->name, $myProjects);
+        $this->assertCount(1, $allProjects);
+        $this->assertArrayHasKey($projectOfLoggedInUser->id, $allProjects);
```

Comment:
here just assert `$this->assertContain($projectOfLoggedInUser->id, allProjects)`